signingscript/docker.d/passwords.yml

```diff
@@ -202,6 +202,12 @@ in:
              {"$eval": "AUTOGRAPH_FENIX_PASSWORD"},
              ["autograph_apk"]
             ]
+        project:mobile:fenix:releng:signing:cert:fennec-nightly-signing:
```

Comment:
It'd be good to fail-fast in case we forgot to set them, so let's add them in [here](https://github.com/mozilla-releng/scriptworker-scripts/blob/master/signingscript/docker.d/init_worker.sh#L147) too.

Reply:
Thanks for the pointer, TIL 🙂👍